Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6-alpine
+
+RUN apk add --update tini
+RUN apk add --update build-base
+RUN apk add --update libffi-dev
+
+ENV PIPENV_VENV_IN_PROJECT=1
+ENV PIPENV_IGNORE_VIRTUALENVS=1
+ENV PIPENV_NOSPIN=1
+ENV PIPENV_HIDE_EMOJIS=1
+
+RUN pip install pipenv
+
+COPY . /
+
+RUN pipenv sync
```

Comment:
would it make sense to use `--bare` for minimal output here?

Reply:
I prefer having a lot of output instead, makes it easier to debug in case of a bad build